docs/guides/admin/docs/upgrade.md

```diff
@@ -23,7 +23,10 @@ or
 
     bin/elasticsearch-plugin install analysis-icu
 
-on all nodes of your Elasticsearch/Opensearch cluster and restart each node.
+on all nodes of your Elasticsearch/Opensearch cluster.
+
+After installation, restart the Elasticsearch/OpenSearch service before starting the Opencast 17 service
```

Comment:
We should decide if it's OpenSearch or Opensearch in our docs. The canonical way would be OpenSearch.

Reply:
The project calls itself OpenSearch. We *should not* decide to change the name. That includes spelling and the case of the letters.

Reply:
I'll file a Pull request for the remaining occurrences of `Opensearch`. It's really just in this file.
Other than that, this seems fine and can be merged.